runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -1342,6 +1347,57 @@ public void testTransformTranslator() throws IOException {
     assertTrue(transform.translated);
   }
 
+  @Test
+  public void testSdkHarnessConfiguration() throws IOException {
```

Comment:
Please also try out a Dataflow GCE job (or existing ITs).

Reply:
The change applies to all existing ITs with use_runner_v2 flag (for example, runner v2 validate runner tests should fail if something goes wrong with this commit). Do you think we need a separate test only for this change?

Reply:
No existing Runner v2 ITs should be adequate. Thanks.